### PR TITLE
task - refactor tests

### PR DESCRIPTION
**Pull request by joefitter**
- bumped so-acceptance to 3.0.0
- moved acceptance tests to apps
- removed prereqs from a acceptance tests

### CODE REVIEW

**Comment by bendiggle** [comment]
@joefitter build is failing!

**Comment by joefitter** [comment]
@bendiggle its getting an old version of so-acceptance - don't think we are cleaning drone cache properly

File: apps/request-declaration/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 module.exports = {
+  name: 'request-declaration',
```
Comment (bendiggle) [inline apps/request-declaration/index.js:4]: if there is `baseUrl` do you need `name`? I thought they did similar things

Comment (joefitter) [reply apps/request-declaration/index.js:4]: yeah we need both - name is the name of the service for session stuff and is always needed, baseUrl is only needed if there is more than 1 route (like here). There will be no baseUrl if omitted - as per UKVI

**Comment by bendiggle** [comment]
@joefitter The build has passed but the tests failed with errors around the page-objects. `Error: Object of type applicantsFullNamePage is not defined in container`. Do we know if this has been happening on the others?

**Comment by joefitter** [comment]
Ah, I think I know what the issue is...

**Comment by joefitter** [comment]
yeah its passing now!

**Comment by bendiggle** [comment]
LGTM Although I'm still not convinced about moving the tests into apps. I like the idea of having to go to one location to the tests. But I can see why you've done it

**Comment by joefitter** [comment]
I'm kind of thinking of apps as apps, as in separate applications which are bundled with their own tests, so everything for that particular app is located in the app folder, almost like a component. Also it simplifies the structure when referring to common steps, their tests will be in the common folder etc.

If we were to carry on down the route of keeping everything in acceptance tests, we would have to mirror the full application structure in acceptance tests, including common components, and IMO it is another place to change things, at least this way the structure is only defined in one place.

I am open to suggestions though! @bendiggle @gxxm @easternbloc ?

**Comment by gxxm** [comment]
I have no problem with the way it is.
:thumbsup: :thumbsup:

**Pull request merged**